make/targets/golang/verify-update.mk

```diff
@@ -8,7 +8,7 @@ chunk_size :=1000
 verify-gofmt:
 	$(info Running `$(GOFMT) $(GOFMT_FLAGS)` on $(go_files_count) file(s).)
 	@TMP=$$( mktemp ); \
-	echo $(GO_FILES) | xargs -n $(chunk_size) $(GOFMT) $(GOFMT_FLAGS) | tee $${TMP}; \
+	find . -name '*.go' -not -path '*/vendor/*' -not -path '*/_output/*' -print | xargs -n $(chunk_size) $(GOFMT) $(GOFMT_FLAGS) | tee $${TMP}; \
```

Comment:
Making this change to use the same `find` command as is executed to populate `GO_FILES` because I couldn't find another way to reliably pipe the contents to the `xargs` process without it counting as arguments and exceeding the max arguments size.